test/integration/forgotten-password.ft.test.js

```diff
@@ -64,7 +64,7 @@ describe('forgotten_password_controller', function () {
     })
   })
 
-  it('send render the page with errors if adminusers returns a 404', async () => {
+  it('show the password reset email sent page if account is not found in adminusers', async () => {
```

Comment:
Should we maybe have a test if there’s something we do expect to show an error for (e.g. adminusers returns a 500)?

Reply:
sure, i'll add one